src/EbmlElement.cpp

```diff
@@ -83,7 +83,7 @@ int CodedSizeLength(uint64 Length, unsigned int SizeLength, bool bSizeIsFinite)
     CodedSize = SizeLength;
   }
 
-  return CodedSize;
+  return int(CodedSize);
```

Comment:
Please use C++-style casts, not C-style casts for the whole commit. Thanks.

Reply:
That is a C++ cast FWIW. But I know what you mean.

Reply:
Ugh, you're right of course. Sorry, my fail, but yeah, I meant `static_cast`s.

Reply:
Done.